src/status_im/subs/wallet/add_account/import_private_key.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im.subs.wallet.add-account.import-private-key
```

Comment:
I like the separation but I think for consistency the other devs input would be helpful here to see if this is an approach we want to take or if we have something else in mind? 🤔